app/src/main/java/vn/mbm/phimp/me/image/CropImage.java

```diff
@@ -120,6 +122,24 @@
 	static Context ctx;
 	static String p[] = null;
     private String newpath;
+
+    @Override
+    public void onBackPressed() {
```

Comment:
put the function below in the file. Although it won't make any difference but it should be below in hierarchy.